acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/tls/BasicTlsOptions.java

```diff
@@ -34,12 +34,12 @@ public BasicTlsOptions(
   }
 
   @Override
-  public File getKeyStoreFile() {
+  public File getStoreFile() {
     return keyStoreFile;
```

Comment:
i guess the other option (continuing from above) would be to go with getKeyFile, and getKeyPasswordFile - they'd read more easily and still give the meaning if you're looking to shorten...